Mathlib/Data/Set/Basic.lean

```diff
@@ -2081,6 +2059,16 @@ theorem diff_singleton_sSubset {s : Set α} {a : α} : s \ {a} ⊂ s ↔ a ∈ s
   sdiff_le.lt_iff_ne.trans <| sdiff_eq_left.not.trans <| by simp
 #align set.diff_singleton_ssubset Set.diff_singleton_sSubset
 
+theorem subset_insert_iff {s t : Set α} {x : α} :
```

Comment:
The `Finset` version aligns with `diff_singleton_subset_iff.symm`, not with this lemma.

Reply:
Hi, I'm not sure what you mean by this. What do I need to do to fix this?

Reply:
Sorry for a slow reply there. You're naming this lemma `Set.subset_insert_iff`, but the lemma statement looks quite different than the `Finset` counterpart `Finset.subset_insert_iff`. In fact, the latter corresponds to the symm of `Set.diff_singleton_subset_iff`. I suggest that you rename this lemma something different to reduce user surprise. I don't have a great name in mind, maybe something like `subset_insert_iff_subset_or_mem_and_diff_subset` but that's quite a mouthful.